miso-web/src/main/webapp/scripts/sample_ajax.js

```diff
@@ -44,19 +44,6 @@ window.Parsley.addValidator('sampleAlias', {
 });
 
 var Sample = Sample || {
-  deleteSample: function(sampleId, successfunc) {
-    if (confirm("Are you sure you really want to delete SAM" + sampleId + "? This operation is permanent!")) {
-      Fluxion.doAjax('sampleControllerHelperService', 'deleteSample', {
-        'sampleId': sampleId,
-        'url': ajaxurl
-      }, {
-        'doOnSuccess': function(json) {
-          successfunc();
-        }
-      });
-    }
-  },
-
   removeSampleFromOverview: function(sampleId, overviewId, successfunc) {
     if (confirm("Are you sure you really want to remove SAM" + sampleId + " from overview?")) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this usage of confirm(...). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)